### ISSUE TITLE
Addon Updater 23.08: Installation failed in NVDA 2023.1

### ISSUE THREAD

**Issue by Carlos-EstebanM** [josephsl/addonUpdater#20]
Steps to reproduce:
1.	In an install or portable copy of NVDA 2023.1 without Addon Updater, try to install Addon Updater 23.08.
Pressing yes button in the confirmation dialog, NVDA show the message failed to install add-on from “location of the file\ addonUpdater-23.08.nvda-addon”,” and the button OK.

The following error is in the NVDA log:

`ERROR - addonHandler.installAddonBundle (12:57:53.502) - gui.ExecAndPump(<function installAddonBundle at 0x03CABB70>) (4128):
task 'onInstall' on addon 'addonUpdater' failed
Traceback (most recent call last):
  File "addonHandler\__init__.pyc", line 281, in installAddonBundle
  File "addonHandler\__init__.pyc", line 544, in runInstallTask
  File "C:\Users\carlo\AppData\Roaming\nvda\addons\addonUpdater.pendingInstall\installTasks.py", line 15, in onInstall
    globalPlugins.addonUpdater.addonUtils.updateState["addonStoreNotificationShown"] = False
AttributeError: module 'globalPlugins' has no attribute 'addonUpdater'
ERROR - gui.addonGui.installAddon (12:57:53.640) - MainThread (14520):
Error installing  addon bundle from C:\Users\carlo\Downloads\addonUpdater-23.08.nvda-addon
Traceback (most recent call last):
  File "addonHandler\__init__.pyc", line 281, in installAddonBundle
  File "addonHandler\__init__.pyc", line 544, in runInstallTask
  File "C:\Users\carlo\AppData\Roaming\nvda\addons\addonUpdater.pendingInstall\installTasks.py", line 15, in onInstall
AttributeError: module 'globalPlugins' has no attribute 'addonUpdater'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "gui\addonGui.pyc", line 578, in installAddon
  File "gui\__init__.pyc", line 677, in __init__
  File "gui\__init__.pyc", line 681, in run
  File "addonHandler\__init__.pyc", line 286, in installAddonBundle
addonHandler.AddonError: Installation failed
`
Note: This issue only affects to new installations of the addon.

**Comment by josephsl** [comment]
Hi, one workaround is installing Add-on Updater 23.07 then updating to 23.08. I will release 23.08.1 to resolve this issue. Thanks.

**Comment by josephsl** [comment]
Resolved in Add-on Updater 23.08.1
